src/utils/fs.ts

```diff
@@ -1,9 +1,8 @@
 import { lstat, readdir, readFile, stat, unlink, writeFile } from 'fs'
```

Comment:
I'm unsure why the direct `fs/promises` imports are not used here in this file. Would be nice to get rid of `promisify` if the same can be done with `fs/promises`